api/kyverno/v2alpha1/policy_exception.go

```diff
@@ -0,0 +1,64 @@
+package v2alpha1
+
+import (
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+// +genclient
+// +kubebuilder:object:root=true
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PolicyException declares resources to be excluded from specified policies.
+type CELPolicyException struct {
+	metav1.TypeMeta   `json:",inline,omitempty"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec declares policy exception behaviors.
+	Spec CELPolicyExceptionSpec `json:"spec"`
+}
+
+func (p *CELPolicyException) GetKind() string {
+	return "CELPolicyException"
+}
+
+// Validate implements programmatic validation
+func (p *CELPolicyException) Validate() (errs field.ErrorList) {
+	errs = append(errs, p.Spec.Validate(field.NewPath("spec"))...)
+	return errs
+}
+
+// PolicyExceptionSpec stores policy exception spec
+type CELPolicyExceptionSpec struct {
+	// PolicyNames identifies the policies to which the exception is applied.
+	PolicyNames []string `json:"policyNames"`
```

Comment:
Why not something like `PolicyRef` instead ?

Reply:
Would this CELPolicyException be applied to all other types of policies? If so, may be worth introducing a `PolicyRef` which includes the policy type and its meta.

Reply:
A `PolicyRef` sounds more extensible, something similar to the `IssuerRef` in cert-manager.